auth/src/test/java/com/firebase/ui/auth/ui/ChooseAccountActivityTest.java

```diff
@@ -1,150 +0,0 @@
-
-import static junit.framework.Assert.assertEquals;
-import static junit.framework.Assert.assertTrue;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-
-@RunWith(CustomRobolectricGradleTestRunner.class)
-@Config(constants = BuildConfig.class, shadows = {ActivityHelperShadow.class}, sdk = 23)
-public class ChooseAccountActivityTest {
-    private FirebaseApp mFirebaseApp;
-    @Mock private CredentialsAPI mCredentialsAPI;
-    @Mock private FirebaseAuth mFirebaseAuth;
-    @Mock private ActivityHelper mActivityHelper;
-
-    @Before
-    public void setUp() {
-        MockitoAnnotations.initMocks(this);
-        mFirebaseApp = TestHelper.initializeApp(RuntimeEnvironment.application);
-        PlayServicesHelper.sApiAvailability = TestHelper.makeMockGoogleApiAvailability();
-
-        when(mCredentialsAPI.isPlayServicesAvailable()).thenReturn(true);
-        when(mCredentialsAPI.isCredentialsAvailable()).thenReturn(true);
-        when(mCredentialsAPI.isAutoSignInAvailable()).thenReturn(true);
-    }
-
-    private Intent createStartIntent() {
-        return AuthUI.getInstance(mFirebaseApp)
-                .createSignInIntentBuilder()
-                .setProviders(AuthUI.EMAIL_PROVIDER, AuthUI.GOOGLE_PROVIDER)
-                .setIsSmartLockEnabled(true)
-                .build(RuntimeEnvironment.application);
-    }
-
-    @Test
-    public void testAutoSignInWithSavedUsernameAndPassword_signsIn() {
-        Intent startIntent = createStartIntent();
-        ChooseAccountActivity chooseAccountActivity =
-                Robolectric.buildActivity(ChooseAccountActivity.class)
-                        .withIntent(startIntent)
-                        .create()
-                        .get();
-
-        when(mCredentialsAPI.getEmailFromCredential()).thenReturn(TestConstants.EMAIL);
-        when(mCredentialsAPI.getPasswordFromCredential()).thenReturn(TestConstants.PASSWORD);
-        when(mCredentialsAPI.getAccountTypeFromCredential()).thenReturn(
-                EmailAuthProvider.PROVIDER_ID);
-
-        when(mActivityHelper.getFirebaseAuth()).thenReturn(mFirebaseAuth);
-        when(mActivityHelper.getFlowParams()).thenReturn(
-                (FlowParameters) startIntent.getParcelableExtra(ExtraConstants.EXTRA_FLOW_PARAMS));
-
-        when(mFirebaseAuth.signInWithEmailAndPassword(
-                TestConstants.EMAIL,
-                TestConstants.PASSWORD))
-                .thenReturn(
-                        new AutoCompleteTask<AuthResult>(
-                                new FakeAuthResult(mock(FirebaseUser.class)), true, null));
-
-        chooseAccountActivity.onCredentialsApiConnected(mCredentialsAPI, mActivityHelper);
-
-        verify(mFirebaseAuth).signInWithEmailAndPassword(
-                TestConstants.EMAIL,
-                TestConstants.PASSWORD);
-
-        assertTrue(Shadows.shadowOf(chooseAccountActivity).isFinishing());
```

Comment:
This test was checking that the email/password survived `signInWithEmailAndPassword()`.